### DISCUSSION TITLE
silent install extras/vcredist

### DISCUSSION THREAD

**Discussion by smathema** [ScoopInstaller/Scoop#5526] [Help and Questions]
Hey all, I'm trying to silently install vcredist using `scoop install extras/vcredist` but unable to do so. The vcredist installation starts installations silently but when it comes to the dependency vcredist2010, the post_install script for it (both x64 and x86) pops up a UI, is there a way bypass this? Appreciate any help!

**Comment by rashil2000** [comment]
There's no way to bypass the UI, sorry!
